python-package/xgboost/dask.py

```diff
@@ -800,9 +814,9 @@ def dispatched_get_shape(worker_id):
         '''Get shape of data in each worker.'''
         LOGGER.info('Get shape on %d', worker_id)
         worker = distributed_get_worker()
-        list_of_parts = _get_worker_x_ordered(worker_map,
+        list_of_parts = _get_worker_parts_ordered(False, worker_map,
                                               partition_order, worker)
```

Comment:
Please help keeping the format.

Reply:
I wonder why pylint did not catch the whitespace error.